core/ui/Components/VisibleTransclude.tid

```diff
@@ -30,13 +30,13 @@ Block transclusions are shown in red, and inline transclusions are shown in gree
 				<!-- Look for a parameter starting with $ to determine if we are in legacy mode -->
 				<$list filter="[<@params>jsonindexes[]] :filter[<currentTiddler>prefix[$]] +[limit[1]]" variable="ignore" emptyMessage="""
 					<!-- Legacy mode: we render the transclusion without a dollar sign for recursionMarker and mode -->
-					<$genesis $type="$transclude" $remappable="no" $names="[<@params>jsonindexes[]]" $values="[<@params>jsonindexes[]] :map[<@params>jsonget<currentTiddler>]" recursionMarker="no" mode=<<mode>>>
+					<$genesis $type="$transclude" $remappable="no" $names="[<@params>jsonindexes[]]" $values="[<@params>jsonindexes[]] :map[<@params>jsonget<currentTiddler>]" recursionMarker="no" mode=<<mode>> $$slotignore="yes">
 						<!-- Reach back up to the grandparent transclusion to get the correct slot value -->
 						<$slot $name="ts-raw" $depth="2"/>
```

Comment:
I think the `$depth` parameter is no longer needed here or below. Here is a test case to illustrate:

```
<$let test="""<$transclude $tiddler="non existing tiddler">abc123</$transclude>""">

<$transclude $variable="test"/>

<$importvariables filter="$:/core/ui/VisibleTransclude">

<$transclude $variable="test"/>
</$importvariables>
</$let>
```

Reply:
Thanks @btheado I've fixed that.